src/ios/GeolocationPlugin.h

```diff
@@ -0,0 +1,42 @@
+//
+//  GeolocationPlugin.h
+//  Geolocation
+//
+//  Created by dev on 2/9/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import <Cordova/CDVPlugin.h>
+#import <CoreLocation/CoreLocation.h>
+#import "sqlite3.h"
+
+@interface GeolocationPlugin : CDVPlugin<CLLocationManagerDelegate>{
+    sqlite3 *contactDB;
+    NSString *dataBasePath;
+    CLLocation* location;
+    NSDateFormatter *DateFormatter;
+    NSString *EnterRegion;
+    NSString *JSONString;
+    NSMutableArray *JSONArray;
+    int m_timeStamp;
+    NSDate* now;
+    NSDate* now1;
+    NSString* coord;
+    BOOL loopflag;
+}
+@property(strong, nonatomic) CLLocationManager *locationManager;
+@property(strong, nonatomic) CLCircularRegion *geoRegion;
+@property(nonatomic, strong) NSThread *thread;
+
+-(void)CreateDataBase;
```

Comment:
method names should always be start with `lowerLettersAndCapitalizedWords` see the [coding guidelines](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html#//apple_ref/doc/uid/20001282-BCIGIJJF). 

this is minor functionality wise but it's a minus on overall code quality